samples/scaleout/distributor-upgrade/sample.md

```diff
@@ -70,6 +70,10 @@ In Versions 6 and above there is a new API for enlisting with the distributor. T
 
 snippet:Enlisting
 
+The last parameter, `capacity` is the maximum number of messages in-flight between the distributor and the worker. The worker sends an acknowledgement (ACK) message (also called `ready` message) for each message processed. The distributor keeps track of these ACKs to ensure no worker is flooded with work that it can't process
+
+Prior to Version 6 the capacity was always equal to the maximum concurrency setting. That caused the worker queue to be empty most of the time because there could never be more messages in-flight than threads ready to process them on the worker side. In Version 6 these settings are decoupled. The snippet above shows how to achieve exactly same behavior as in previous versions.
```

Comment:
>  That caused the worker queue to be empty most of the time because there could never be more messages in-flight than threads ready to process them on the worker side.

Do we really want to say that? Sounds like a bad thing the first time you read it :) Also, it's not obvious for me why that cuases worker queue to be empty?

> Prior to Version 6 the capacity was always equal to the maximum concurrency setting. That caused the worker queue to be empty most of the time because there could never be more messages in-flight than threads ready to process them on the worker side. In Version 6 these settings are decoupled. The snippet above shows how to achieve exactly same behavior as in previous versions.

In Versions 6 and below the capacity was always set to the same value as the maximum concurrency setting, which means there were never more messages in-flight than threads ready to process them on the worker side.  (clarify the empty thing here)

A new setting was introduced in Version 6, which allows to explicitly control the capacity value. The snippet above shows how to set it to the same value as in previous versions.